Azure-Samples/azure-maps-fullscreen-control
# Invert value of fullscreenchanged flag

## Purpose
Fixes #1 - Flag attached to the fullscreenchanged event does not represent the state of the screen

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
Add the control and listen to the event. Verify that the boolean flag represents the state of the screen
